### ISSUE TITLE
test: dropout layer probability parameter

### ISSUE THREAD

**Issue by sperezacuna** [sperezacuna/oracle-challenge-f2#4]
From [doc](https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html):

> p ([float] – probability of an element to be zeroed. Default: 0.5

Now set to 0.3
